docs/release/upgrading_to_v2_0.md

```diff
@@ -39,7 +39,17 @@ To do this, the `LD_PRELOAD` setting must be passed to all tasks
 of the MPI-parallel job, for example with
 `mpirun -genv LD_PRELOAD /usr/lib64/libioil.so -np 8 -f hostfile dcp $SOURCE $DEST`.
 
-When upgrading to DAOS Version 2.0, please also verify the supported
+When planning the upgrade to DAOS Version 2.0, please also verify the supported
 operating system levels as outlined in the
 [DAOS Version 2.0 Support](./support_matrix_v2_0.md) document.
 If an OS upgrade is required, this should be performed prior to upgrading DAOS.
+
+!!! note
```

Comment:
Lower case _n_ on _note_?

Reply:
yes the "!!!! note" is part of our stylesheet, to produce a box with this note when published on daos.io.